cmd/integration_test/cmd/run.go

```diff
@@ -62,7 +63,8 @@ func init() {
 	runE2ECmd.Flags().StringP(instanceProfileFlagName, "i", "", "IAM instance profile name to attach to ec2 instances")
 	runE2ECmd.Flags().StringP(subnetIdFlagName, "n", "", "EC2 subnet ID")
 	runE2ECmd.Flags().StringP(regexFlagName, "r", "", "Run only those tests and examples matching the regular expression. Equivalent to go test -run")
-	runE2ECmd.Flags().IntP(maxInstancesFlagName, "m", 1, "Run tests in parallel with multiple EC2 instances")
+	runE2ECmd.Flags().IntP(maxInstancesFlagName, "m", 1, "Run tests in parallel on same instance within the max EC2 instance count")
+	runE2ECmd.Flags().IntP(maxConcurrentTestsFlagName, "c", 1, "Maximum number of parallel tests that can be run at a time")
```

Comment:
This is exactly what `maxInstancesFlagName` was supposed to do
I'm totally ok changing the name, but do we need both?